src/shared/Internal.Shared.Tasks/Internal.Shared.Tasks.proj

```diff
@@ -0,0 +1,43 @@
+﻿<Project>
```

Comment:
Shared projects require the VS SDK which would make it impossible to build this project on cross-plat. Just using regular source includes for this.